ladder/teams/sig-policy.yaml

```diff
@@ -1,8 +1,8 @@
 members:
+- bimmlerd
 - christarazi
 - derailed
 - doniacld
-- fristonio
```

Comment:
Is this intentional? Deepesh is not in the Emeritus list

Reply:
cc @fristonio I haven't seen a PR from this contributor for some time, and I think it makes sense for us to keep the list trimmed down to people actively maintaining the code area. If and when @fristonio returns to actively contributing, we can re-evaluate adding them back into the team.